app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,10 @@
+class DashboardController < ApplicationController
+  
+  def index
+    @jobs = Job.all
+    @companies = Company.all
+    @jobs_by_level_of_interest_count = Company.jobs_by_level_of_interest(@jobs)
```

Comment:
if we have to pass in a collection of jobs here, why not create this in the `Job` model?